crates/storage/store_db/libmdbx.rs

```diff
@@ -683,6 +685,83 @@ impl StoreEngine for Store {
             .take(MAX_SNAPSHOT_READS);
         Ok(iter.collect::<Vec<_>>())
     }
+
+    fn __add_block(&self, block: Block, receipts: Vec<Receipt>) -> Result<(), StoreError> {
```

Comment:
not a huge fan, but for consistency accross the codebase this should probably be named add_block_inner